github/apps.go

```diff
@@ -5,14 +5,65 @@
 
 package github
 
-import "context"
+import (
+	"context"
+	"fmt"
+	"time"
+)
 
 // AppsService provides access to the installation related functions
 // in the GitHub API.
 //
 // GitHub API docs: https://developer.github.com/v3/apps/
 type AppsService service
 
+// App represents a GitHub App.
+type App struct {
+	ID          *int       `json:"id,omitempty"`
```

Comment:
This is actually more of a question for @shurcooL or @willnorris...

In the light of #597, do you think we should start using `int64` for numeric IDs for new APIs in order to make the implementation of #597 easier (by providing an example and reducing the number of changes needed)?

Or do you think that we should remain internally consistent and just change everything all at once when #597 is addressed?

Reply:
IMO it's unclear what the state of #597 is right now. The issue has been opened a while ago, but I haven't seen any reports of issues from production use. I'm not convinced that we should act right away, or how we should act. I think that we need to collect more data in that issue (or, wait until there's a concrete need to act).

Because of that, I think we should not try to incorporate that issue into this one. So, I suggest we stay consistent, and deal with `int` -> `int64` changes (or another solution), if at all, later and separately from this PR.